schema/compose-spec.json

```diff
@@ -918,9 +918,17 @@
           "type": "object",
           "patternProperties": {
             ".+": {
-              "type": ["string", "array"]
-            },
-            "uniqueItems": false
+              "oneOf": [
+                {
+                  "type": "string"
+                },
+                {
+                  "type": "array",
+                  "items": {},
```

Comment:
when set with an array, items are actually of type string. Schema should reflect this constraint

Reply:
I've added this constraint now